client/components/Help/Help.js

```diff
@@ -2,37 +2,74 @@ import React, { Component, PropTypes } from 'react';
 import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
 import { connect } from 'react-redux';
 import { NewProblem } from '../NewProblem/NewProblem';
-import { Problem } from './../Problem/Problem';
+import { Problem } from '../Problem/Problem';
+import { AnswersList } from '../AnswersList/AnswersList';
 import { getProblems, addProblem } from '../../actions/actions';
 import './Help.scss';
 
 class Help extends Component {
     state = {
-        isNewProblemVisible: false
+        isNewProblemVisible: false,
+        isAnswersListVisible: false
     };
 
-    componentDidMount() {
-        this.props.dispatch(getProblems());
-    }
-
     newButtonClick = () => {
-        this.setState({
-            isNewProblemVisible: true
-        });
+        if (this.state.isAnswersListVisible) {
+            this.setState({
+                isAnswersListVisible: false
+            });
+            setTimeout(() => {
+                this.setState({
+                    isNewProblemVisible: true
+                });
+            }, 500);
+        } else {
+            this.setState({
+                isNewProblemVisible: true
+            });
+        }
     };
 
     addButtonClick = (problem) => {
-        this.props.dispatch(addProblem(problem));
+        this.props.addProblem(problem);
         this.setState({
             isNewProblemVisible: false
         });
     };
 
+    viewButtonClick = () => {
+        if (this.state.isNewProblemVisible) {
+            this.setState({
+                isNewProblemVisible: false
+            });
+            setTimeout(() => {
+                this.setState({
+                    isAnswersListVisible: true
+                });
+            }, 500);
+        } else {
+            this.setState({
+                isAnswersListVisible: true
+            });
+        }
+    };
+
+    hideButtonClick = () => {
+        this.setState({
+            isAnswersListVisible: false
+        });
+    };
+
+    componentDidMount() {
+        this.props.getProblems();
+    }
+
     render() {
         const { isNewProblemVisible } = this.state;
+        const { isAnswersListVisible } = this.state;
```

Comment:
const { isNewProblemVisible } = this.state;
const { isAnswersListVisible } = this.state;
=>
const { isNewProblemVisible, isAnswersListVisible } = this.state;